### PR TITLE
Bug/my location tests

### PR DESCRIPTION
**Pull request by cpittet**
This is a small PR that add tests for the "my location" feature.

The tests do not provide a full coverage.

NOTE : the tests are failing because of the GoogleMap helper, which is already fixed by @Mathematou. So this PR must be merged after his.

Thanks for your review and comments :)

### CODE REVIEW

File: app/src/androidTest/java/com/github/sdpteam15/polyevents/MyLocationTests.kt [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+            public override fun matchesSafely(view: View): Boolean {
+                val parent = view.parent
+                return parent is ViewGroup && parentMatcher.matches(parent)
+                        && view == parent.getChildAt(position)
+            }
+        }
+    }
+
+    private fun denyPermissions() {
+        if (Build.VERSION.SDK_INT >= 23) {
```
Comment (Cell00phane) [inline app/src/androidTest/java/com/github/sdpteam15/polyevents/MyLocationTests.kt:139]: Check not needed anymore, we've upgraded the min sdk version to 26 :)

**Review by Cell00phane — APPROVED** [review]
This should improve the test coverage so nice. Not sure what's going on in these tests, maybe @Mathematou could take a look since he's working on that part as well.

File: app/src/androidTest/java/com/github/sdpteam15/polyevents/MyLocationTests.kt (hunk 1 above)
Comment (cpittet) [reply app/src/androidTest/java/com/github/sdpteam15/polyevents/MyLocationTests.kt:139]: Oups, you're right. This was something I took from somewhere else and did not check, thanks :)

Comment (cpittet) [reply app/src/androidTest/java/com/github/sdpteam15/polyevents/MyLocationTests.kt:139]: And actually there is a bug in this part, I'm looking into it.

**Review by AlessioVerardo — APPROVED** [review]
Stephane says everything. Just need to make the tests pass now :)

**Pull request merged**